EventSubscriber/ProductSerializerSubscriber.php

```diff
@@ -73,9 +73,10 @@ public function onPostSerialize(ObjectEvent $event): void
         $visitor = $event->getVisitor();
         $visitor->setData('extension', [/* TODO seo and excerpt */]);
         $visitor->setData('urls', [/* TODO localized urls */]);
-        $visitor->setData('product', $object);
-        $visitor->setData('content', $this->resolveContent($structure, $object->getContentData()));
-        $visitor->setData('view', $this->resolveView($structure, $object->getContentData()));
+        $visitor->setData('product', $object->getProduct());
+        $visitor->setData('productInformation', $object->getProductInformation());
+        $visitor->setData('content', $this->resolveContent($structure, $object->getContent()->getData()));
+        $visitor->setData('view', $this->resolveView($structure, $object->getContent()->getData()));
```

Comment:
`id`, `code`, `template` is missing when the serializer config of `ProductView` is empty

Reply:
Currently it's the same behavior like before?

Reply:
the pages also provides this properties on the root level